src/encoded/schemas/long_read_rna_quantification_quality_metric.json

```diff
@@ -0,0 +1,41 @@
+
+{
+    "title": "long-read RNA-seq quantification quality metric",
+    "description": "Schema for reporting long-read RNA-seq pipeline quantification quality metric",
+    "id": "/profiles/long_read_rna_quantification_quality_metric.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["step_run", "award", "lab", "genes_detected"],
+    "additionalProperties": false,
+    "identifyingProperties": ["uuid"],
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "quality_metric.json#/properties" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/submitted"},
+        { "$ref": "mixins.json#/aliases"},
+        { "$ref": "mixins.json#/attribution" },
+        { "$ref": "mixins.json#/standard_status"},
+        { "$ref": "mixins.json#/assay" }
+   ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "genes_detected": {
+            "title": "Genes detected",
+            "description": "Number of GENCODE genes detected for the replicate",
+            "type": "integer",
+            "minimum": 1
```

Comment:
Could this be 0?

Reply:
I don't see why not, unless @ottojolanki is opposed